tests/Microsoft.Identity.Test.Unit/PublicApiTests/PublicClientApplicationTests.cs

```diff
@@ -856,6 +856,9 @@ public async Task AcquireTokenSilent_EmptyLoginHint_TestAsync()
         /// <returns></returns>
         [TestMethod]
         [DeploymentItem(@"Resources\TokenCacheWithWamId.json")]
+        //https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3859
+        //https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3375
+        [Ignore("This test no longer possible since MSAL doesnt return local tokens in acquireTokenSilent anymore.")]
```

Comment:
If you are ignoring the test, why do you want to keep the code?

Reply:
Agreed, delete it.